### PR TITLE
[Refactor](executor)Add information_schema.workload_groups

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Add information_schema.workload_groups.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -2296,15 +2297,16 @@ public TFrontendPingFrontendResult ping(TFrontendPingFrontendRequest request) th
...
-                return MetadataGenerator.getSchemaTableData(request);
-            default:
-                break;
+        if (!request.isSetSchemaTableName()) {
+            return MetadataGenerator.errorResult("Fetch schema table name is not set");
+        }
+        // tvf queries
+        if (request.getSchemaTableName() == TSchemaTableName.METADATA_TABLE) {
+            return MetadataGenerator.getMetadataTable(request);
+        } else {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:2306]: change to  else if (request.getSchemaTableName() == TSchemaTableName.INFORMATION_SCHEMA) {
            
        } else {
                throw new TException("xxxxx")
        }

Then we could make sure if we add another ENUM, the logic will not be wrong.

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by morningman — APPROVED** [review]

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**